@joelverhagen I have trouble compiling `NuGet.Commands` for netstandard1.3. Getting a lot of "are you missing an assembly reference?"
```
home/tmds/repos/NuGet.Client/src/NuGet.Core/NuGet.Commands/RestoreCommand/ContentFiles/FileProviderGlobbingFile.cs(5,17): error CS0234: The type or namespace name 'AspNet' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)
/home/tmds/repos/NuGet.Client/src/NuGet.Core/NuGet.Commands/RestoreCommand/ContentFiles/FileProviderGlobbingFile.cs(6,28): error CS0234: The type or namespace name 'FileSystemGlobbing' does not exist in the namespace 'Microsoft.Extensions' (are you missing an assembly reference?)
/home/tmds/repos/NuGet.Client/src/NuGet.Core/NuGet.Commands/RestoreCommand/ContentFiles/FileProviderGlobbingDirectory.cs(6,17): error CS0234: The type or namespace name 'AspNet' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)
/home/tmds/repos/NuGet.Client/src/NuGet.Core/NuGet.Commands/RestoreCommand/ContentFiles/FileProviderGlobbingDirectory.cs(7,28): error CS0234: The type or namespace name 'FileSystemGlobbing' does not exist in the namespace 'Microsoft.Extensions' (are you missing an assembly reference?)
/home/tmds/repos/NuGet.Client/src/NuGet.Core/NuGet.Commands/RestoreCommand/ContentFiles/VirtualFileProvider.cs(4,17): error CS0234: The type or namespace name 'AspNet' does not exist in the namespace 'Microsoft' (are you missing an assembly reference?)
...
```

--- a/src/NuGet.Core/NuGet.Common/PathUtil/DirectoryUtility.cs
+++ b/src/NuGet.Core/NuGet.Common/PathUtil/DirectoryUtility.cs
@@ -22,6 +22,11 @@ namespace NuGet.Common
             else
             {
                 path = Path.GetFullPath(path);
+                if (Directory.Exists(path))
+                {
+                    return;
+                }
+                // ensure directories exists starting from the root
                 var root = Path.GetPathRoot(path);
                 var sepPos = root.Length - 1;
                 do

--- a/test/NuGet.Core.Tests/NuGet.Common.Test/DirectoryUtilityTests.cs
+++ b/test/NuGet.Core.Tests/NuGet.Common.Test/DirectoryUtilityTests.cs
@@ -62,9 +62,16 @@ namespace NuGet.Common.Test
                 RedirectStandardOutput = true,
                 RedirectStandardInput = true,
                 UseShellExecute = false,
-                Arguments = "-c %a " + path,
                 FileName = "stat"
             };
+            if (RuntimeEnvironmentHelper.IsLinux)
+            {
+                startInfo.Arguments = "-c %a " + path;
+            }
+            else
+            {
+                startInfo.Arguments = "-f %A " + path;
+            }
 
             using (Process process = new Process())
             {

